src/components/tabs/FeatureThresholdsTab.tsx

```diff
@@ -209,11 +209,10 @@ export default function FeatureThresholdsTab(inputProps: FeatureThresholdsTabPro
     props.onChange(newThresholds);
   };
 
-  const onCategoricalThresholdChanged = (index: number, enabled_categories: boolean[]): void => {
-    const newThreshold = { ...props.featureThresholds[index], enabled_categories };
+  const onCategoricalThresholdChanged = (index: number, enabledCategories: boolean[]): void => {
+    const newThreshold = { ...props.featureThresholds[index], enabledCategories };
```

Comment:
This is a bug that is easy to have happen.  We could have linter enforce these var naming conventions, but also if newThreshold were typed it might have given a warning about a bad property name too

Reply:
I'm embarrassed that I didn't catch this when I made the earlier fix, I should have slowed down to make sure that I didn't break categorical features too 😶 That's a good idea, thank you!